### DISCUSSION TITLE
IV infusion dosing Dataset with multiple occasions for each patient

### DISCUSSION THREAD

**Discussion by MissLesley2** [nlmixr2/nlmixr2#158] [Q&A]
Hi,

I have a dataset with multiple occasions for each patient for Monolix like

![Screen Shot 2023-06-09 at 12 55 25 PM](https://github.com/nlmixr2/nlmixr2/assets/133399263/940da7c7-914a-4764-a7d2-eafbcd92287b)

It is infusion administration with dose amount and infusion duration.
Could you please let me know whether it is right to covert this dataset into nlmxir2 format like below?
![Screen Shot 2023-06-09 at 12 56 05 PM](https://github.com/nlmixr2/nlmixr2/assets/133399263/218d26e3-d68b-4c53-b7c3-a0afed328010)

or
![Screen Shot 2023-06-09 at 12 56 33 PM](https://github.com/nlmixr2/nlmixr2/assets/133399263/77727e8e-9c53-40bf-8061-1a76f3f59284)

Is nlmxir2 able to calculate inter-occasion variability? If not, how can I add it into nlmxir2?

Thanks,
Lesley

**Comment by mattfidler** [comment]
Hi Lesley,

There is not any inter-occasion variability support at this time.

For the first example, it should work, but EVID should be `0` for observations.

The second example should work too, though it isn't typically used.

You could also use the `Dur` column with the more traditional evid `1` or `0`

**Reply by mattfidler** [reply]
I converted this to a discussion because it is more of a question.
